peps/pep-0728.rst

```diff
@@ -578,17 +590,19 @@ arguments of this type when constructed by calling the class object::
 Interaction with Mapping[KT, VT]
 --------------------------------
 
-A TypedDict type can be assignable to ``Mapping[KT, VT]`` types other than
-``Mapping[str, object]`` as long as all value types of the items on the
-TypedDict type is :term:`typing:assignable` to ``VT``. This is an extension of this
+A TypedDict type is assignable to a type of the form ``Mapping[str, VT]``
+when all value types of the items in the TypedDict
+are :term:`typing:assignable` to ``VT``. For the purpose of this rule, a
+TypedDict that does not have ``extra_items=`` or ``closed=`` set is considered
+to have an item with a value of type ``object``. This is an extension of this
 assignability rule from the `typing spec
 <https://typing.readthedocs.io/en/latest/spec/typeddict.html#assignability>`__:
 
-    * A TypedDict with all ``int`` values is not :term:`typing:assignable` to
+    * A (non-closed) TypedDict with all ``int`` values is not :term:`typing:assignable` to
```

Comment:
Maybe use square brackets here because this modifies a part of a quote?

Reply:
Oh, I didn't realize this was a quote. Since the PEP's text should be treated as an amendment to the spec, I think it's better to rewrite to avoid this.